src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,33 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeptions.InvalidUserException;
+import core.basesyntax.exeptions.RegisterServiceException;
 import core.basesyntax.model.User;
+import core.basesyntax.validators.UserForRegistrationValidator;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final String INVALID_USER_MESSAGE = "User is invalid: ";
+    private static final String USER_ALREADY_EXISTS_MESSAGE = "User with given login "
+            + "is already exists";
+    private final StorageDao storageDao;
+
+    public RegistrationServiceImpl() {
+        storageDao = new StorageDaoImpl();
+    }
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RegisterServiceException {
```

Comment:
```suggestion
    public User register(User user) {
```
You already handled this exception in method